shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -26,4 +29,27 @@ - (void)touchesBegan:(NSSet*)touches withEvent:(UIEvent*)event {
   }
 }
 
+- (void)applicationDidEnterBackground:(UIApplication *)application {
+#if FLUTTER_RUNTIME_MODE == FLUTTER_RUNTIME_MODE_DEBUG
+  // The following keeps the Flutter session alive when the device screen locks
+  // in debug mode. It allows continued use of features like hot reload and 
+  // taking screenshots once the device unlocks again.
+  //
+  // Note the name is not an identifier and multiple instances can exist. 
+  debugBackgroundTask = [application beginBackgroundTaskWithName:@"Flutter debug task"
+                                               expirationHandler:^{
+      FTL_DLOG(WARNING) << "\nThe OS has terminated the Flutter debug connection for being "
```

Comment:
You need `FTL_LOG`. The `DLOG` variant will only be available in unoptimized builds. We release optimized builds.